core/mixins/_media-queries.scss

```diff
@@ -97,3 +105,16 @@
     @content;
   }
 }
+
+
+/**
+ * Retina.
+ */
+
+@mixin respond-retina($resolution: $retina-resolution) {
+  $resolution: if(unitless($resolution), $resolution * 96dpi, $resolution);
+  
+  @media (min-resolution: $resolution) {
+    @content;
+  }
+}
```

Comment:
> So I gave this a spin and I'd expect to not have to pass in a parameter when I just want to use the Scally default ($retina-resolution). But that throws an compilation error.

This works for me. Have you `@import`ed the _retina-resolution.scss as per the README? https://github.com/chris-pearce/scally/pull/127/files#diff-04c6e90faac2675aa89e2176d2eec7d8R414